keyboards/the_royal/liminal/keymaps/default/keymap.c

```diff
@@ -0,0 +1,165 @@
+    ),
+
+        /* RAISE layer
+        * ┌─────┬────┬────┬────┬────┬────┬────┬────┬────┬────┬────┬────┬────┬──────┐
+        * │  \  │    │    │    │    │    │    │    │    │    │    │    │ =  │      │
+        * ├─────┼────┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴──────┤
+        * │  |  │CAPS │    │    │    │    │    │    │    │    │    │    │    +     │
+        * ├─────┼─────┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬────────┤
+        * │     │       │    │    │    │    │    │    │    │    │ [  │ ]  │        │
+        * ├─────┼────┬──┴─┬──┴─┬──┴────┴────┴────┴────┴────┴───┬┴────┴┬───┴─┬──────┤
+        * │     │    │    │    │                               │      │     │      │
+        * └─────┴────┴────┴────┴───────────────────────────────┴──────┴─────┴──────┘
+     */
+    [_RAISE] = LAYOUT_base_kit_ansi(
+        KC_BSLS,  XXXXX,  XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,  KC_EQUAL,  XXXXX,
+        KC_PIPE, KC_CAPS,  XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,     KC_PLUS,
+        XXXXX,  XXXXX,     XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,   KC_LBRC,    KC_RBRC,  XXXXX,
+        XXXXX,  XXXXX, XXXXX, XXXXX,                            XXXXX,                    XXXXX, XXXXX, XXXXX
+    ),
+
+        /* OTHER layer
+        * ┌─────┬────┬────┬────┬────┬────┬────┬────┬────┬────┬────┬────┬────┬──────┐
+        * │HOME │    │    │    │    │    │    │    │    │    │    │ UP │    │      │
+        * ├─────┼────┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴┬───┴──────┤
+        * │END  │     │    │    │    │    │    │    │    │    │LEFT│RGHT│          │
+        * ├─────┼─────┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬──┴─┬────────┤
+        * │     │       │    │    │    │    │    │    │    │    │DOWN│    │        │
+        * ├─────┼────┬──┴─┬──┴─┬──┴────┴────┴────┴────┴────┴───┬┴────┴┬───┴─┬──────┤
+        * │RESET│    │    │    │                               │      │     │      │
+        * └─────┴────┴────┴────┴───────────────────────────────┴──────┴─────┴──────┘
+     */
+    [_OTHER] = LAYOUT_base_kit_ansi(
+        XXXXX,  XXXXX,  XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    KC_UP,  XXXXX,  XXXXX,
+        XXXXX,  XXXXX,  XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    KC_LEFT,    KC_RIGHT,     XXXXX,
+        XXXXX,  XXXXX,     XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,    XXXXX,   KC_DOWN,    XXXXX,  XXXXX,
+        TD(TD_RST),  XXXXX,      XXXXX,      XXXXX,                            XXXXX,                    XXXXX,       XXXXX,       XXXXX
+    )
+
+};
+
+uint16_t get_tapping_term(uint16_t keycode) {
+  switch (keycode) {
+    case LT(_LOWER, KC_SPACE):
+      return 160;
+    case TD(TD_DBQT):
```

Comment:
Just a heads up, this won't actually work, at this time.   Not until #6259 is merged.

Reply:
Its working right now....322155342 lol Did I mess something up

Reply:
or you mean the timing?

Reply:
Oh okay I see.  Should I remove it or will it just be default until #6259 is merged?

Reply:
It should be fine to leave in here.